### PR TITLE
Create ui-c8y-1019-22-0-block-searching-when-creating-editing-private-smartrule-5817-graft-release-cd.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-55913](https://cumulocity.atlassian.net/browse/MTM-55913)
Original PR: [6198](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6198)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1019-22-0-block-searching-when-creating-editing-private-smartrule-5817-graft-release-cd.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: block searching when creating editing private smartrule (#5817) [GRAFT][release/cd] (#6198)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1019-22-0-block-searching-when-creating-editing-private-smartrule-5817-graft-release-cd.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Block searching when creating or editing a private smart rule
```

File: content/change-logs/application-enablement/ui-c8y-1019-22-0-block-searching-when-creating-editing-private-smartrule-5817-graft-release-cd.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-55913
+version: 1019.22.0
+---
+block searching when creating editing private smartrule (#5817) [GRAFT][release/cd] (#6198)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1019-22-0-block-searching-when-creating-editing-private-smartrule-5817-graft-release-cd.md:17]: The Claude AI suggested to use the following body:
```suggestion
In the past, it was possible to search for devices when creating or editing a private smart rule, which could potentially leak information about devices the user should not have access to. To address this security concern, searching for devices is now blocked when creating or editing a private smart rule. This change ensures that users can only select devices they are authorized to access, preventing any potential data leaks. The impact of this change is limited to the creation and editing of private smart rules, and users can still search for devices when creating or editing non-private smart rules as before.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by BeateRixen** [comment]
@janhommes Please review and approve

**Review by BeateRixen — APPROVED** [review]

**Review by pmich-sag — APPROVED** [review]

**Pull request merged**